uui/components/navigation/MainMenu/MainMenuButton.tsx

```diff
@@ -3,7 +3,10 @@ import { Button, ButtonProps } from '@epam/uui-components';
 import { IAdaptiveItem, withMods } from '@epam/uui-core';
 import { ReactComponent as SvgTriangle } from '../../../icons/chevron-down-24.svg';
 
-export interface MainMenuButtonMods {
+interface MainMenuButtonMods {
+    /*
+    * Defines component type.
```

Comment:
Need to clarify what is the difference between 'primary' and 'secondary'

Reply:
fixed